model/testing/src/icon4py/model/testing/definitions.py

```diff
@@ -49,19 +49,11 @@ class Grid:
     description: str
     kind: GridKind
     sizes: Mapping[Literal["cell", "edge", "vertex"], int]
-    file_name: str | None = None
-    uri: str | None = None
```

Comment:
would not `GridDescription` be a better name for this?

Reply:
Absolutely! Will do the change.